docs/getting_started/setup.md

```diff
@@ -96,11 +96,16 @@ cd build
 cmake ..
 make
 ```
-Running Cmake will create `crow_all.h` file and place it in the build directory.
+Running Cmake will create `crow_all.h` file and place it in the build directory.<br>
+
+You can run tests with following commands:
+```
+ctest -V
+```
 
 ##Installing Crow
 
-if you wish to use crow globally without copying `crow_all.h` in your projects, you can install `crow` on your machine by using `make install` command instead of `make` after `cmake..`. Thus, the procedure will look like
+if you wish to use crow globally without copying `crow_all.h` in your projects, you can install `Crow` on your machine with the following procedure below.
```

Comment:
please remove the ``.
Also "the following procedure" and "the procedure below" mean the same thing, you can remove one